common/test/acceptance/tests/lms/test_lms_courseware.py

```diff
@@ -806,6 +806,29 @@ def test_perform_problem_reset_and_navigate(self):
         problem1_content_after_coming_back = self.problem_page.problem_content
         self.assertEqual(problem1_content_before_switch, problem1_content_after_coming_back)
 
+    def test_perform_problem_reset_and_focus(self):
```

Comment:
Instead of putting this test in test_lms_courseware.py, let's consolidate all the problem tests in test_problem_types.